src/schemas/validateRegisterSchema.ts

```diff
@@ -0,0 +1,20 @@
+import * as yup from "yup";
+
+export const validateRegisterSchema = yup.object().shape({
+  email: yup
+    .string()
+    .required("Email required")
```

Comment:
Will be better to use a friendly message to the user like email can't be empty

Reply:
done